src/requests/apis.js

```diff
@@ -2,7 +2,7 @@ let travelFetch = {
   dashboardInfo(userId) {
     return Promise.all([
       fetch(`https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${userId}`),
```

Comment:
Possible suggestion if you're tired of looking at long URLs: you could store the repeating parts of your fetch URLs in a variable & interpolate it in your fetch requests